client/my-sites/marketplace/pages/marketplace-plugin-upload-status/index.tsx

```diff
@@ -163,6 +193,15 @@ const MarketplacePluginInstall = ( { productSlug } ): JSX.Element => {
 								: `/plugins/${ productSlug }/${ selectedSiteSlug }`
 						}
 					/>
+				) : nonInstallablePlanError ? (
+					<EmptyContent
+						illustration="/calypso/images/illustrations/error.svg"
+						title={ translate(
+							"Your current plan doesn't allow plugin installation. Please upgrade to Business plan first."
+						) }
+						action={ translate( 'Upgrade to Business plan' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **25** times:
`translate( 'Upgrade to Business Plan' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/guard-plugin-install-route)